models/DataObject/ClassDefinition/Data/Objectbricks.php

```diff
@@ -965,4 +958,26 @@ public static function collectCalculatedValueItems($container, &$list = [])
             }
         }
     }
+
+    public function getParameterTypeDeclaration(): ?string
+    {
+        return '?\\' . Objectbrick::class;
+    }
+
+    public function getReturnTypeDeclaration(): ?string
+    {
+        return '?\\' . Objectbrick::class;
+    }
+
+    public function getPhpdocInputType(): ?string
+    {
+        return '\\' . Objectbrick::class;
```

Comment:
The parameter type is nullable and should match.
```suggestion
        return '\\' . Objectbrick::class . '|null';
```